docker/jdk8/x86_64/ubuntu/Dockerfile-openj9

```diff
@@ -0,0 +1,38 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM openjdk_container
+
+MAINTAINER Mark Stoodley <mstoodle@ca.ibm.com>
```

Comment:
I assume @mstoodle is aware that you are naming him as the maintainer?!
We should consider a community e-mail address to redirect any contact made via this address to the team, rather than burdening an individual.

Reply:
The hazards of copy, paste, and edit. Can we use a placeholder until we get a group email, or do we need an actual person?

...Do we need specific file maintainer at all?

Reply:
I am ok with my name being used for the moment. The other Dockerfiles at the repo are maintained by @tellison and @gdams, so I'm in very good company ;) .

Nonetheless, I think it does make sense to have a more community focused maintenance story.

Reply:
I've opened #188 for this.  It's certainly not a blocker for this PR.